### DISCUSSION TITLE
Standard Name Table Release - v85, May 2024

### DISCUSSION THREAD

**Discussion by efisher008** [cf-convention/discuss#319] [Announcements relevant to CF]
Dear All,

The standard name table has been updated. The current version is now version **85**, dated **21 May 2024**. The changes have been published on the NERC vocabulary server and a KWIC index for version 85 has also been created. All changes are listed below.

Matthias Schneider, Dawn Petraitis, Bo Liu, Marie Peniguel, Fiona O'Connor and Grigory Nikulin have been added to the [list of standard name contributors](http://cfconventions.org/Data/cf-standard-names/docs/standard-name-contributors.html).

Best wishes,
Ellie

P.S. I am using the Announcements category on Discussions for this release, as this is a new functionality the CF community is now utilising and the category name would suggest this is a very appropriate place for this message.

**V85 standard name changes:**

`New names`

a. [#130](https://github.com/cf-convention/discuss/issues/130)
Names proposed by Markus Fiebig re/ particle optical properties. 

b. [#199](https://github.com/cf-convention/discuss/issues/199)
Names proposed by Matthias Schneider re. averaging kernels for mole fraction profiles of atmospheric trace gases.

c. [#222](https://github.com/cf-convention/discuss/issues/222)
Names proposed by Aurore Biardeau re. Ekman current.

d. [#261](https://github.com/cf-convention/discuss/issues/261)
Names proposed by Dawn Petraitis re. spectral wave data.

e. [#274](https://github.com/cf-convention/discuss/issues/274)
Names proposed by Swati Gehlot and Bo Liu re. bio-geo-chemical tracer variables for Project PalMod.

f. [#280](https://github.com/cf-convention/discuss/issues/280)
Names proposed by Marie Peniguel re. wave energy flux.

g. [#299](https://github.com/cf-convention/discuss/issues/299)
Names proposed by Fiona O'Connor re. flux diagnostic calls to an radiative forcing model using a reference ozone field.

h. [#303](https://github.com/cf-convention/discuss/issues/303)
Name proposed by Grigory Nikulin re. depth to bedrock.

`New aliases (term changes)`

a. [#186](https://github.com/cf-convention/discuss/issues/186) 
Two further names from the original proposal aliased to include mention of radiative flux.
Changes proposed by Jonathan Gregory.

`volume_extinction_coefficient_in_air_due_to_ambient_aerosol_particles` -> `volume_extinction_coefficient_of_radiative_flux_in_air_due_to_ambient_aerosol_particles`

`volume_extinction_coefficient_in_air_due_to_cloud_particles` -> `volume_extinction_coefficient_of_radiative_flux_in_air_due_to_cloud_particles`

`Updated descriptions`

a. [#271](https://github.com/cf-convention/discuss/issues/271)
Modified description of` atmosphere_stability_k_index`.
Change proposed by Jonathan Gregory.

b. [#270](https://github.com/cf-convention/discuss/issues/270)
A group of temperature-based standard name descriptions altered to add clarifying text regarding use of the units_metadata CF attribute for on-scale vs. difference units. 
Changes proposed by Jonathan Gregory.

**Comment by JonathanGregory** [comment]
Ellie @efisher008 wrote

> I am using the Announcements category on Discussions for this release, as this is a new functionality the CF community is now utilising and the category name would suggest this is a very appropriate place for this message.

I agree, this is definitely an announcement. I have labelled it `vocabulary`. At the moment, the choice of labels is confusing because the discussions are in the `discuss` repo, where we have issue for standard names. I expect we will be able to untangle this soon.

**Comment by japamment** [comment]
Closing this to tidy up previous announcements.
